### PR TITLE
EWB-465 Add name and name type

### PR DESCRIPTION
**Pull request by var-ram**

### CODE REVIEW

**Comment by var-ram** [comment]
Task linked: [EWB-465 Add Names to evolve-grpc](https://app.clickup.com/t/6929263/EWB-465)

File: proto/zepben/protobuf/cim/iec61970/base/core/Name.proto [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  /**
+   * Any free text that name the object.
+   */
+  string name = 1;
+
+  /**
+   * Type of this name.
+   */
+  NameType type = 2;
```
Comment (gcarwardine) [inline proto/zepben/protobuf/cim/iec61970/base/core/Name.proto:34]: I don't think this should be a NameType. It means every identified object that comes back with a name will be sent with a full NameType. We don't want this. This hsould probably just be the name type name.

**Review by gcarwardine — CHANGES_REQUESTED** [review]

**Review by gcarwardine — APPROVED** [review]

**Comment by gcarwardine** [comment]
I'm now happy with this in principle @var-ram. However let's not merge this until you flesh out some of the implementation in the SDK. You can obviously install locally to develop against. Are you happy with that?

**Comment by var-ram** [comment]
> I'm now happy with this in principle @var-ram. However let's not merge this until you flesh out some of the implementation in the SDK. You can obviously install locally to develop against. Are you happy with that?

Oh yes. That's what I had in mind. 👍

**Pull request merged**